modules/ossm-installing-operator.adoc

```diff
@@ -6,6 +6,11 @@
 = Installing the {SMProductShortName} Operator
 :context: ossm-installing-operator
 
+[WARNING]
+====
+For clusters without existing {SMProduct} instances, follow this procedure. {SMProduct} operates cluster-wide, requiring proper scope configuration to prevent conflicts between {istio} control planes. For clusters with {SMProduct} 3 or later, see xref:../install/ossm-deploying-multiple-service-meshes-on-single-clusterc#ossm-deploying-multiple-service-meshes-on-single-cluster[Deploying multiple service meshes on a single cluster].
```

Comment:
xref is not working here. Is it expected?

Reply:
It's still not working for me in the preview.

Reply:
@FilipB Give me some time to look into this issue. It is probably a preview error.

Reply:
@FilipB It's fixed. PTAL. :)